src/redux/create-module.js

```diff
@@ -89,35 +101,55 @@ function createModule(namespace, createDefinition) {
     eventErrors.map(err => `> ${err}`).join('\n')
   )
 
+  // ======================================================
+  // Finish Module
+  // ======================================================
+
+  // ----------------------------------
+  // Constants
+  // ----------------------------------
   // Interface conformance checks complete, begin generating the module
   const generateConstant = (eventType) => `@@${namespace}/${eventType}`
   generatedModule.constants = eventTypes.reduce((acc, eventType) => {
     acc[eventType] = generateConstant(eventType)
     return acc
   }, {})
 
+  // ----------------------------------
+  // Reducer
+  // ----------------------------------
   // Generate reducer based on provided event handlers.
   generatedModule.reducer = createReducer(
     mapKeys(generateConstant, definition.handlers),
     definition.initialState
   )
 
+  // ----------------------------------
+  // Events
+  // ----------------------------------
   // Generate event creators that automatically apply a namespaced `type`
   // property to the action object.
   generatedModule.events = eventTypes.reduce((acc, eventType) => {
     const namespacedEventType = generatedModule.constants[eventType]
     const liftedEventCreator = definition.events[eventType]
+
     function generatedEventCreator(...args) {
       const liftedResult = liftedEventCreator(...args)
       return { ...liftedResult, type: namespacedEventType }
     }
+
     acc[eventType] = generatedEventCreator
     return acc
   })
 
-  // Expose selectors and actions, no transformations needed.
-  generatedModule.selectors = definition.selectors
-  generatedModule.actions = definition.actions
+  // ----------------------------------
+  // Others
+  // ----------------------------------
+  // Expose keys where no transformations are needed.
+  generatedModule.initialState = { ...definition.initialState }
+  generatedModule.handlers = { ...definition.handlers }
```

Comment:
Here is where I've exposed these.  I also copied the others so as to not allow mutation.